package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymesh-subquery",
-  "version": "5.5.0",
```

Comment:
Maybe a master -> alpha is needed?

Reply:
Not needed. My idea with package.json version was to us for migrations and whenever we think we need some migrations then only its necessary to bump this up to next possible release version. I did it here just to make it sync with latest version.